polaris-react/.storybook/preview-body.html

```diff
@@ -1,5 +0,0 @@
-<style>
-  .sb-show-main.sb-main-padded {
-    padding: 0;
-  }
-</style>
```

Comment:
@kyledurand this felt like something that could be cleaned up. Let me know what you think about this PR.